library/alloc/src/vec.rs

```diff
@@ -171,7 +171,7 @@ use crate::raw_vec::RawVec;
 ///
 /// // ... and that's all!
 /// // you can also do it like this:
-/// let x : &[usize] = &v;
+/// let u: &[usize] = &v; // or &[_]
```

Comment:
I think this would be better as code, not a comment. Also, is there a reason you changed the name?
```suggestion
/// let x: &[usize] = &v;
/// // or like this:
/// let y: &[_] = &v;
```

Reply:
Because `u` is usually used as an alternative to `v` for vec but `x` feels weird to be used as vec.